sw/host/opentitanlib/src/io/jtag.rs

```diff
@@ -23,6 +23,9 @@ pub struct JtagParams {
 
     #[arg(long, default_value = "1000")]
     pub adapter_speed_khz: u64,
+
+    #[arg(long, default_value = "false")]
```

Comment:
I am not sure about the default value. We should definitely consider whether this is the right value for JTAG tests to enable debugging in CI. Maybe we want to explicitely enable logging in those tests. @nbdd0121 might have some opinion.

Reply:
Do we need the logs in CI? if a test is failing, it seems you would want to check for reproducibility locally, in which case you would enable the flag locally. Silencing the verbosity of the openocd logs in CI should save CI runtime to (admittedly on a little thought).